gamestonk_terminal/brokers/README.md

```diff
@@ -0,0 +1,160 @@
+# BROKERS
+
+This page shows the available brokers for loading your holdings.  If yours is not listed, please submit a new issue and we will look into adding it.
+
+Current brokers:
+* rh : Robinhood
+* alp : Alpaca
+* ally: Ally
+* dg : Degiro
+
```

Comment:
Anyone looked into adding Interactive Brokers?

Reply:
#343.  Not free for api